### PR TITLE
[Premium Files Preview] Made the following cmdlets changes

### PR DESCRIPTION
**Pull request by arkrasik**
- Set-PowerBIDataset that initiates abf <-> premium files migration
- Get-PowerBIWorkspaceMigrationStatus to check migrations statuses

### CODE REVIEW

File: src/Modules/Data/Commands.Data/SetPowerBIDataset.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+        [Parameter(Mandatory = true, ParameterSetName = DatasetNameParameterSetName)]
+        [Parameter(Mandatory = true, ParameterSetName = ObjectNameParameterSetName)]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = DatasetIdParameterSetName)]
+        [Parameter(Mandatory = true, ParameterSetName = DatasetNameParameterSetName)]
+        [Parameter(Mandatory = true, ParameterSetName = ObjectIdParameterSetName)]
+        [Parameter(Mandatory = true, ParameterSetName = ObjectNameParameterSetName)]
+        [ValidateSet("Abf", "PremiumFiles")]
```
Comment (CodeCyclone) [inline src/Modules/Data/Commands.Data/SetPowerBIDataset.cs]: ValidateSet won't be needed as you are using an enum

File: src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspaceMigrationStatus.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+### Example 1
+```powershell
+PS C:\> Get-PowerBIWorkspaceMigrationStatus -Id "3244f1c1-01cf-457f-9383-6035e4950fdc"
+```
+
+## PARAMETERS
+
+### -All
+{{Fill All Description}}
```
Comment (CodeCyclone) [inline src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspaceMigrationStatus.md]: Needs to be filled in

File: src/Modules/Data/Commands.Data/SetPowerBIDataset.cs (hunk 1 above)
Comment (arkrasik) [reply src/Modules/Data/Commands.Data/SetPowerBIDataset.cs]: no; there is unknown in enum, don't want it

File: src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspaceMigrationStatus.md (hunk 2 above)
Comment (arkrasik) [reply src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspaceMigrationStatus.md]: oops; missed; ty

File: src/Modules/Data/Commands.Data/GetPowerBIDataset.cs [hunk 3]
```diff
@@ -53,6 +53,15 @@ public class GetPowerBIDataset : PowerBIClientCmdlet, IUserScope, IUserFilter, I
...
         public int? First { get; set; }
 
+        [Alias("Expand")]
+        [Parameter(Mandatory = false, ParameterSetName = IdParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = NameParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ListParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ObjectIdParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ObjectNameParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ObjectListParameterSetName)]
+        public string Include { get; set; }
```
Comment (CodeCyclone) [inline src/Modules/Data/Commands.Data/GetPowerBIDataset.cs]: Either add a ValidateSet or do an array of enums here

Comment (arkrasik) [reply src/Modules/Data/Commands.Data/GetPowerBIDataset.cs]: we'll start with actualStorage and add other later

**Pull request closed without merging**